Automattic/wp-calypso
# Stats: Adding navigation arrows for subscribers

Related to #75935 

## Proposed Changes

* extending Subscribers chart with navigation arrows to swap visible dates (still using mock data)
* introducing default `quantity` values for different periods to avoid pulling the same number of points for different periods

## Testing Instructions



* Open the live branch for this PR
* Navigate to the traffic stats page at `/stats/day/:siteSlug`
* Add the query string `?flags=stats/subscribers-section` to enable the subscriber counts page
* Verify that arrows are visible next to period selection and they change visible periods (use X axis label to verify this)

## Pre-merge Checklist
